src/test/scala/com/mwz/sonar/scala/scalastyle/ScalastyleSensorSpec.scala

```diff
@@ -232,10 +233,55 @@ class ScalastyleSensorSpec
     context.setActiveRules(activeRules)
 
     val result: ActiveRule = ScalastyleSensor.ruleFromStyleError(context, styleError).value
-    result.language === "scala"
-    result.internalKey === "rule1"
-    result.severity === ErrorLevel.toString
-    result.params.asScala === Map("param1" -> "value1")
+    result.language should ===("scala")
```

Comment:
It's a minor thing, but if you could omit the redundant parentheses in your tests, so we could be more consistent with the testing style in the project, that would be great.

Reply:
They are required if we want to use the `===` operator here. I will switch to `shouldEqual` so that we can avoid the parentheses.